pkg/podgrouper/podgrouper/podgrouper.go

```diff
@@ -29,6 +29,8 @@ type Interface interface {
 }
 
 type podGrouper struct {
+	queueLabelKey  string
```

Comment:
Dont see any usage of having it as field of poGrouper

Reply:
Good catch, removed.